script.js

```diff
@@ -93,12 +93,12 @@ let deletePost = async function (domNode) {
     let confirmDialogs = [];
     i = 0;
     while (confirmDialogs.length < 1) {
-        await delay(1000);
+        await delay(3000);
```

Comment:
The delay of 1000 works for me. Could this be because you are farther away from the data center?